great_expectations/execution_engine/split_and_sample/pandas_data_sampler.py

```diff
@@ -0,0 +1,84 @@
+import hashlib
+import random
+
+import pandas as pd
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.core.id_dict import BatchSpec
+from great_expectations.execution_engine.split_and_sample.data_sampler import (
+    DataSampler,
+)
+
+
+class PandasDataSampler(DataSampler):
+    """Methods for sampling a pandas dataframe."""
+
+    def sample_using_limit(
+        self, df: pd.DataFrame, batch_spec: BatchSpec
+    ) -> pd.DataFrame:
+        """Sample the first n rows of data.
+
+        Args:
+            df: pandas dataframe.
+            batch_spec: BatchSpec with sampling_kwargs for limit sampling e.g. sampling_kwargs={"n": 100}.
+
+        Returns:
+            Pandas dataframe reduced to number of items specified in BatchSpec sampling kwargs.
+        """
+        self.verify_batch_spec_sampling_kwargs_exists(batch_spec)
+        self.verify_batch_spec_sampling_kwargs_key_exists("n", batch_spec)
+
+        n: int = batch_spec["sampling_kwargs"]["n"]
+        return df.head(n)
+
+    @staticmethod
```

Comment:
The methods from here down are just moved from the execution engine. I can clean them up in a follow-on PR.